.dockerignore

```diff
@@ -2,3 +2,17 @@
 deps/*
 doc
 ebin
+AUTHORS
+CONTRIBUTING
+CONTRIBUTORS
+build.cmd
+Dockerfile
+Dockerfile.*
+LICENSE
+Makefile
+prepare-release.sh
+*.md
+start.*
+TRANSLATORS
+zotonic_install
+priv/emacs
```

Comment:
Better to invert this: specify the dirs we *don’t* want to ignore, using `!...`.